recipes/cogent3/meta.yaml

```diff
@@ -16,17 +17,19 @@ build:
 
 requirements:
   host:
-    - python >=3
+    - python >=3.8, <3.12
```

Comment:
why do you restrict this to 3.12?

Reply:
Because of the dependence of cogent3 on `numba`, there is some delay in which versions of Python can be supported.